//! try & catch in JS

//! try
// the try statement allows you to define a block of code to be tested for errors while it being executed.

//! catch
// the catch statement allows you to define a block of code to be executed, if an error occurs in try block.

// Example 1
let x = 5;
try {
  console.log(x);
} catch (error) {
  console.log(error);
}

// Example 2
console.log("hello");
console.log("hello");
console.log("hello");
try {
  console.log(a);
} catch (error) {
  console.log(error);
}
console.log("hello");
console.log("hello");
console.log("hello");
